src/helpers/validationUtils.ts

```diff
@@ -23,4 +24,14 @@ const validateCountry = (country: string): boolean => {
 
 const emailRegex = /^([\w.+-]+)@([\w-]+\.)+([\w]{2,})$/gm;
 
-export { validateCountryCode, validateCountry, emailRegex };
+/**
+ * Returns valid list of seasons by removing untraditional entries provided by TMDB
+ * I.E. 'Specials' and unreleased seasons
+ * @param seasons
+ * @returns {Season[]}
+ */
+const validateSeasons = (seasons: Season[]): Season[] => {
```

Comment:
I don't think I would consider this a validation function. Maybe just semantics, but IMO a validation function should generally return a boolean, valid or invalid. I would rename this to `removeInvalidSeasons` or something like that.

Reply:
By the same logic, does it make more sense to place this function in another file? I considered placing it in `showFilterUtils` and maybe naming it `filterValidSeasons` or something along those lines, but I thought that may be out of scope of the file's purpose.

@Thenlie